### DISCUSSION TITLE
Doesnt connect wirelessly IP

### DISCUSSION THREAD

**Discussion by NekoSuneVR** [Sergey004/Quest2-VRC#22] [General]
it doesnt connect wirelessly without been cable plugged in PC. i did all ADB connections and fully connected. but app doesnt see wireless and forcing use link cable on checks.

not sure pose do that, but pose be wireless supported too.

i normally use wireless connection since my PC doesnt keep my headset full charged. even since has type-c port supported

**Comment by Sergey004** [comment]
Wireless adb cannot be installed without running adb tcpip 5555, sorry, but this is how Wireless adb works on Quset (it works in Meta Quest Developer Hub).

**Comment by Sergey004** [comment]
Fixed in 62e59e040a381fe3ab4c4f0fc6dacf26f851a8ab

**Comment by NekoSuneVR** [comment]
ZeroConf doesnt work, only old way working and wont start ADB or connect by ADB.

all getting is failed connect ADB error no other errors.

**Comment by Sergey004** [comment]
Have you tried installing an addon for this? (And try update headset firmware to v55)

**Comment by NekoSuneVR** [comment]
i check if my headset is v55 but apk already installed

**Comment by NekoSuneVR** [comment]
partly my update is v54 and no update for v55

**Comment by Sergey004** [comment]
> i check if my headset is v55 but apk already installed

Have you run the app on the headset at least once?

**Comment by NekoSuneVR** [comment]
yes but when run it auto closes

**Comment by Sergey004** [comment]
Well you can try this as alternative https://github.com/thedroidgeek/oculus-wireless-adb

**Comment by NekoSuneVR** [comment]
i managed got update v55 but still doesnt work on app

**Comment by NekoSuneVR** [comment]
but havent tried that other app yet

**Comment by Sergey004** [comment]
> i managed got update v55 but still doesnt work on app

I'll try to fix my app, I reinstalled it and it too started to close after launching

**Comment by Sergey004** [comment]
I rebuilt the app, hope it works https://github.com/Sergey004/Quest2-VRC/releases/tag/New_Addon

**Comment by Sergey004** [comment]
Reopen if you need

**Comment by Sergey004** [comment]
Now is 100% working even on non Quest device
https://github.com/Sergey004/Quest2-VRC/releases/tag/Finaly
